package.json

```diff
@@ -34,7 +34,8 @@
     "test:lint:js": "eslint --max-warnings=0 ./lib ./test ./features-legacy ./bin ./features ./doc/build-error-codes.js ./plugins/available/functional-test-plugin",
     "test:lint:js:fix": "eslint --max-warnings=0 --fix ./lib ./test ./features-legacy ./bin ./features ./doc/build-error-codes.js",
     "doc-error-codes": "node -r ts-node/register doc/build-error-codes",
-    "docker:install": "docker-compose run kuzzle_node_1 npm install"
+    "docker:install": "docker-compose run kuzzle_node_1 npm install",
+    "docker:run": "docker-compose run kuzzle_node_1 npm run --"
```

Comment:
The naming is not very explicit since this script is running NPM and not the project

Reply:
Yes, I agree, I had not idea about naming but maybe something like this could be more explicit: `docker:script` `docker:npm` ?

Reply:
`docker:npm` seem fine to me :+1: